src/components/crown/crownButtons/crownButton.vue

```diff
@@ -41,7 +41,7 @@ export default {
   display: flex;
 }
 img, i {
```

Comment:
One last minor change: for `<i>` elements you should use font-size instead height. Notice the following screenshot, the pintor palette have a height of 16px:
![Screen Shot 2022-08-17 at 16 59 35](https://user-images.githubusercontent.com/90727999/185231710-a576d538-f5ed-481d-afd5-433ab4462a3e.png)